project-canvas/src/components/login/Login.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { ColorSchemeProvider } from "@mantine/core"
+import { render, screen } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { Login } from "./Login"
+
+jest.mock("./jira-cloud/loginToJiraCloud.ts", () => ({
+  loginToJiraCloud: jest.fn(),
+}))
+jest.mock("electron", () => ({
+  ipcRenderer: {
+    send: jest.fn(),
+  },
+}))
+
+jest.mock("./jira-server/loginToJiraServer.ts", () => ({
+  loginToJiraServer: jest.fn(),
+}))
+const mockedUsedNavigate = jest.fn()
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useNavigate: () => mockedUsedNavigate,
+}))
+
+jest.mock("react-i18next", () => ({
+  useTranslation: () => ({
+    t: (str: string) => str,
+    i18n: {
+      changeLanguage: () => new Promise(() => {}),
+    },
+  }),
+  initReactI18next: {
+    type: "3rdParty",
+    init: () => {},
+  },
+}))
+
+describe("<Login />", () => {
+  it(" providerLogin should be set to JiraCloud after clicking on the Jira Cloud Button", async () => {
```

Comment:
```suggestion
  it("should be set to JiraCloud after clicking on the Jira Cloud Button", async () => {
```
following it come `should ...`